device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,327 @@
+{
+    "chassis": {
+        "name": "Mellanox-SN2700",
```

Comment:
Per @keboliu, 'name' should be `MSN2700`

Reply:
fixed